file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,588 @@
+      "! Position number
+      position_number    TYPE ty_check_position_number,
+      "! <p class="shorttext">Check Execution Sequence</p>
+      "! Check execution sequence
+      execution_sequence TYPE ty_execution_sequence,
+      "! <p class="shorttext">Check Execution Control</p>
+      "! Check execution control
+      execution_control  TYPE ty_execution_control,
+      "! <p class="shorttext">Selection Filters on Check</p>
+      "! Selection filters on check
+      check_filter       TYPE ty_check_filters,
+    END OF ty_check_attribute.
+  "! <p class="shorttext">Check Attributes</p>
+  "! Check attributes
+  TYPES ty_check_attributes TYPE SORTED TABLE OF ty_check_attribute WITH UNIQUE KEY check_id.
+  TYPES:
+    "! <p class="shorttext">Representation Type</p>
+    "! Assigned representation type
+    BEGIN OF ty_representation_type,
+      "! <p class="shorttext">Assigned Representation Type</p>
+      "! Assigned representation type
+      representation_type TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_representation_type.
+  "! <p class="shorttext">Representation Types</p>
+  "! Assigned representation types
+  TYPES ty_representation_types TYPE SORTED TABLE OF ty_representation_type WITH UNIQUE KEY representation_type.
+  TYPES:
+    "! <p class="shorttext">Comparison Type</p>
+    "! Comparison type
+    BEGIN OF ty_comparison_type,
+      "! <p class="shorttext">Comparison Type</p>
+      "! Comparison type
+      "! $required
+      comparison_type      TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Comparison Type Description</p>
+      "! Description of the comparison type
+      "! $required
+      description          TYPE ty_long_description,
+      "! <p class="shorttext">Position Number</p>
+      "! Position number
+      "! $required
+      position_number      TYPE ty_position_number,
+      "! <p class="shorttext">Implementing Class</p>
+      "! Implementing Class
+      "! $required
+      class_name           TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Data Source</p>
+      "! Data Source
+      "! $required
+      data_source          TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Existence Check</p>
+      "! Assigned existence check
+      "! $required
+      existence_check      TYPE ty_existence_check,
+      "! <p class="shorttext">Status Checks</p>
+      "! Assigned status checks
+      status_check         TYPE ty_status_checks,
+      "! <p class="shorttext">Content Checks</p>
+      "! Assigned content checks
+      content_check        TYPE ty_content_checks,
+      "! <p class="shorttext">Check Attributes</p>
+      "! Check attributes
+      "! $required
+      check_attributes     TYPE ty_check_attributes,
+      "! <p class="shorttext">Representation Types</p>
+      "! Assigned representation types
+      "! $required
+      representation_types TYPE ty_representation_types,
```

Comment:
These Representation Types have been already part in Content Checks Comparism Field Groups. Is there a reason why they are here twice?